flyway-commandline/src/main/java/org/flywaydb/commandline/Main.java

```diff
@@ -254,7 +254,7 @@ private static void printUsage() {
         LOG.info("baselineVersion              : Version to tag schema with when executing baseline");
         LOG.info("baselineDescription          : Description to tag schema with when executing baseline");
         LOG.info("baselineOnMigrate            : Baseline on migrate against uninitialized non-empty schema");
-        LOG.info("configFile                   : Config file to use (default: ./flyway.conf)");
+        LOG.info("configFile                   : Config file to use (default: <install-dir>/conf/flyway.conf)");
```

Comment:
I'm specifying the the `<install-dir>`, otherwise it looks like it looks for a `conf/flyway.conf` relative to the working directory you execute the `flyway` CLI in.

Technically, it loads `<install-dir>/conf/flyway.conf`, `<user-home>/flyway.conf` and `<current-dir>/flyway.conf` by default, but I believe the important change is removing mentions of `flyway.properties`